recipes/conterminator/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1-8e9a7" %}
+
+package:
+  name: conterminator 
+  version: {{ version|replace("-", ".")  }}
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/martin-steinegger/conterminator
```

Comment:
@martin-steinegger could you create a real release tarball? This would enable us to also backup the tarball in the long run.

Reply:
I can not link the archive since I use submodules.

Reply:
@bgruening I changed the release to include all dependencies as tarball.